ui/src/message_popup/author/questions_page_test.jsx

```diff
@@ -15,6 +19,16 @@ function testProps(props) {
   };
 }
 
+function withContext(child) {
```

Comment:
Awesome!  This is from `experience_page_test` and seems generally useful, so I'd pull it out into another file and the `test` folder is probably the right home for it.